src/site/_includes/components/updates/updates.njk

```diff
@@ -5,6 +5,8 @@
       <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds-from-desktop">
           <h2 class="govuk-heading-l">Updates</h2>
+          <p class="govuk-body"><span class="govuk-!-font-weight-bold">20 October 2023:</span> The Defence Developer Services (D2S) will shortly commence work on its re-use strategy and the discovery and delivery of a re-use components service. As part of this, we will commence a Discovery activity for the Design System Toolkit. During the Discovery Phase, the MOD.UK Frontend and Design Library will be in maintenance mode with no further enhancements being made during this time. Only high priority issues such as security fixes or outages will be completed.</p>
+          <p class="govuk-body"><u><span class="govuk-!-font-weight-bold govuk-!-font-underlined">Getting support</span></u></br>For high priority issues, you can contact <a href="mailto:design-system@digital.mod.uk">design-system@digital.mod.uk</a>. If you have any general questions, you can contact <a href="mailto:UKStratComDD-Foundry-DSO-Engage@mod.gov.uk">UKStratComDD-Foundry-DSO-Engage@mod.gov.uk</a>.</p>
```

Comment:
This `span` should be an `h3`, so that it _is_ a heading rather than it just _looking like_ a heading. This is called [semantic HTML](https://developer.mozilla.org/en-US/docs/Glossary/Semantics). Giving it the `class` `"govuk-heading-m"` should achieve the visual effect you're after.

Similarly, the [`u` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/u) should be avoided. More generally the GOV.UK Design System (and thus MOD.UK Frontend) only use underlines for link text, so we should leave it out here.

Reply:
Talked to Rachael about this, the guidance site also has the conflicting contact info - she has said to just remove the `For high priority issues...` sentence